aws.yaml

```diff
@@ -16,7 +16,7 @@
       name: cluster-operator
       provider: aws
       version: 0.22.0
-  date: 2019-10-28T12:00:00Z
+  date: 2019-11-05T12:00:00Z
```

Comment:
I think CI in installations was checking if we don't have 2 releases with the same date?